lib/archethic/db/embedded_impl.ex

```diff
@@ -282,9 +282,9 @@ defmodule Archethic.DB.EmbeddedImpl do
   @doc """
   Return the first address of given chain's address
   """
-  @spec get_first_chain_address(address :: binary()) :: binary()
-  def get_first_chain_address(address) when is_binary(address) do
```

Comment:
If we remove this one, the PR about `get_first_address` might not be possible

Reply:
It's just renaming the function as it is actually getting the genesis address and not the first address

Reply:
Ok

Reply:
We might rename the message itself then , as it might create confusion

Reply:
Done !